kaggle/pipeline.go

```diff
@@ -13,6 +13,91 @@ import (
 	"github.com/d-sparks/gravy/trading"
 )
 
+func ParseOrDie(floatString string) float64 {
+	float, err := strconv.ParseFloat(floatString, 64)
+	gravyutil.FatalIfErr(err)
+	return float
+}
+
+// Assumes the stock is published at dates[lb] and dates[ub] but not between.
+// Linearly interpolate based on y = y1 + ((y2-y1)/(x2-x1))*(x-x1)
+func InterpolateForSymbol(
+	symbol string,
+	lb int,
+	ub int,
+	dates []time.Time,
+	data map[time.Time]*trading.Window,
+) {
+	// If the stock was unlisted for more than 90 days, assume the new ticker is a different
```

Comment:
Is this a valid heuristic?

Reply:
Nah, it's totally made up. I hope our data will eventually be clean enough that we don't need this interpolation at all. Also: Is it possible that two different stocks (in different time periods) would have the same symbol?